onnx2trt_utils.hpp

```diff
@@ -238,6 +238,35 @@ inline int div_ceil(int n, int d) {
   return (n - 1) / d + 1;
 }
 
+// Given an axis and TRT input tensor, modify and return the axis value
+// depending on whether or not the input is a TRT tensor.
+inline Status convert_axis(int& axis, int nbDims, bool isTensor)
+{
+  enum { BATCH_DIM = 0 };
+  // If the input is a TRT tensor, no modification to axis is necessary
```

Comment:
My brain is mush at the moment, but this confuses me. Doesn't it still need to subtract 1 and check that it's not the batch dim?

Reply:
Seems my brain was more mush :boom:. Good catch